store/db/prefixdb_test.go

```
@@ -0,0 +1,49 @@
+
+	mockDB.EXPECT().Delete(gomock.Eq(append(prefix, key...))).Return(nil)
+
+	err = pdb.Delete(key)
+	require.NoError(t, err)
+
+	mockDB.EXPECT().Has(gomock.Eq(append(prefix, key...))).Return(false, nil)
+
+	has, err = pdb.Has(key)
+	require.NoError(t, err)
+	require.False(t, has)
+}
```

golangci-lint (1.62.2)

File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(cosmossdk.io) -s prefix(github.com/cosmos/cosmos-sdk) --custom-order

(gci)